app/importers/planning_applications_importer.rb

```diff
@@ -80,7 +80,7 @@ def with_property(row)
                               code: row[:property_code],
                               type: row[:property_type])
       property.build_address(
-        full: row[:full].blank? ? row[:address] : row[:full],
```

Comment:
Address is missing from the latest data so there's no fallback.
Even if address were present there will still be cases when it's null